src/helpers/genericTypeHelper.ts

```diff
@@ -0,0 +1,26 @@
+import { consts, texts } from '../config';
+import { GenericType } from '../types';
+
+export const getGenericItemDetailTitle = (genericType: GenericType): string => {
+  switch (genericType) {
+    case GenericType.Commercial:
+      return texts.commercial.commercial;
+    case GenericType.Job:
+      return texts.job.job;
```

Comment:
this will grow with upcoming generic types, right?

Reply:
yes, this will grow with each new generic type